src/style.css

```diff
@@ -0,0 +1,82 @@
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir-Book.ttf");
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir.ttc");
+  font-style: normal;
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir-Heavy.ttf");
+  font-style: normal;
+  font-weight: 900;
+}
+
+.searchform {
+  box-sizing: border-box;
+  font-weight: 300px;
+  margin-top: 20px;
+}
+
+.searchbar {
+  position: relative;
+  display: flex;
+}
+
+.input {
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  color: #3d4e61;
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input:focus-within {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-weight: 900;
+  color: #3d4e61;
+  text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+.input-small {
+  height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.icon {
+  position: absolute;
+}
+
+.icon-big {
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 25px;
+}
+
+.icon-small {
+  width: 11px;
+  height: 11px;
+  left: 13px;
+  top: 15px;
+}
```

Comment:
see [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) p.2

Reply:
There should be no class "icon-big", just "icon" and "icon-small". 
Fixed.
Thank you for checking my task.